### PR TITLE
Update the bug bounty amount

### PR DESCRIPTION
**Pull request by tomrittervg**
2016-2021 comes to $1.9m and I don't have any data pre 2016.  @dveditz Does $2m sound good or do you want to try to find some more data to pump it higher?

### CODE REVIEW

**Comment by dveditz** [comment]
This version looks OK. Alternatively you could say "over $2m since 2015" and be accurate in a more defined time-frame, with the downside that someone's then going to ask "what about the decade before that?"

Found a 2013 Q1 HoF update with the text
> As of this date, we have paid out $935,000 in bounties. Congratulations to everybody who has participicated!

That was not long after we upped client bounties from $500 to $3000 so it went up faster after that.

There's a sheet called "Aggregate Client and Web Bounty Data 2015 - 2017" shared with you that shows ~$500K for each of those years.

I can't find details for 2014.

found another mail bringing things up to Sept 30, 2015:
>  Grand total since program inception through Sept,30, 2015 = 1,844,979

Not sure how we got such a wonky total if every bounty I ever remember was a round number ¯\_(ツ)_/¯

So that's basically $3.8M with 3 months missing at the end of 2015 (which are in the sheet mentioned above). Should bring us very close to $4M total.

I'm changing my answer to "pump it up higher".

File: bedrock/security/templates/security/bug-bounty/hall-of-fame.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h1 class="mzp-c-article-title">Firefox Bug Bounty Rewards</h1>
   </header>
 
   <p>On behalf of the Mozilla and the millions of people who visit our sites, use Firefox and our other products we would like to thank them for their hard work in helping to make us more secure.</p>
-  <p>As of this date, we have paid out almost <b>$1,000,000</b> across all of our bounties. Congratulations to everybody who has participated!</p>
+  <p>As of this date, we have paid over <b>$2,000,000</b> across all of our bounties. Congratulations to everybody who has participated!</p>
```
Comment (craigcook) [inline bedrock/security/templates/security/bug-bounty/hall-of-fame.html:17]: Based on the comment from @dveditz it sounds like this should increase even more?

 ```suggestion
  <p>As of this date, we have paid over <b>$4,000,000</b> across all of our bounties. Congratulations to everybody who has participated!</p>
```

File: bedrock/security/templates/security/bug-bounty/web-hall-of-fame.html [hunk 2]
```diff
@@ -14,7 +14,7 @@ <h1 class="mzp-c-article-title">Web And Services Bug Bounty Hall of Fame</h1>
   </header>
 
   <p>On behalf of the Mozilla and the millions of people who visit our sites, use Firefox and our other products we would like to thank them for their hard work in helping to make us more secure.</p>
-  <p>As of this date, we have paid out over <b>$1,600,000</b> across all of our bounties. Congratulations to everybody who has participated!</p>
+  <p>As of this date, we have paid out over <b>$2,000,000</b> across all of our bounties. Congratulations to everybody who has participated!</p>
```
Comment (craigcook) [inline bedrock/security/templates/security/bug-bounty/web-hall-of-fame.html:17]:
```suggestion
  <p>As of this date, we have paid out over <b>$4,000,000</b> across all of our bounties. Congratulations to everybody who has participated!</p>
```

**Review by craigcook — CHANGES_REQUESTED** [review]

**Comment by stevejalim** [comment]
Heya @craigcook can this be merged now?

**Comment by craigcook** [comment]
I defer @dveditz on if the $4M amount is accurate.

**Comment by stevejalim** [comment]
@dveditz @tomrittervg Can we merge this, please?

**Comment by stevejalim** [comment]
@tomrittervg All done and merged - cheers!

**Pull request merged**